.travis.yml

```diff
@@ -1,6 +1,17 @@
+sudo: false
+os:
+  - linux
 language: node_js
 node_js:
-  - "0.10"
   - "0.12"
   - "4"
-  - "node"
+  - "6"
+env:
+  matrix:
```

Comment:
This option allows combine all possible combinations of options. [docs on travis](https://docs.travis-ci.com/user/customizing-the-build#Build-Matrix)